vms/platformvm/txs/mempool/mempool.go

```diff
@@ -64,24 +64,26 @@ type Mempool interface {
 	// possibly reissued.
 	MarkDropped(txID ids.ID, reason error)
 	GetDropReason(txID ids.ID) error
+
+	// Iterate iterates over the txs until f returns false
+	Iterate(f func(tx *txs.Tx) bool)
```

Comment:
Does the caller have any guarantee around the order this is done in? If so, we should note it here.

Reply:
This comes from the one of the `gossip` interfaces being implemented, but from the perspective of gossip this ordering can be arbitrary from the caller's pov